change/@ni-nimble-components-c92408f2-14ec-4254-9aac-0babfa7fbbd1.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
> I am addressing issues 3-5. I will check with Brandon whether 1 and 2 are really the designs we want.

Is the result of that captured anywhere / in tech debt or features?

Reply:
Created #2011 and updated description